test/raven.test.js

```diff
@@ -3079,6 +3081,29 @@ describe('Raven (public API)', function() {
       });
     }
 
+    it("should treat Schrodinger's Error in the same way as regular Error", function() {
+      // Schrodinger's Error is an object that is and is not an Error at the same time
```

Comment:
could you characterize the properties of these errors a little bit more explicitly (like, "it has the prototype of an error but also passes isPlanObject because X", or maybe link the the issue? i'm not sure i would understand based only on this test